cidc_api/models.py

```diff
@@ -455,6 +455,14 @@ def trial(cls):
     # The object URI for the raw excel form associated with this upload
     gcs_xlsx_uri = Column(String, nullable=False)
 
+    def alert_upload_success(self):
+        """Send an email notification that an upload has succeeded."""
+        # (import these here to avoid a circular import error)
+        import emails, gcloud_client
```

Comment:
why are you using an import here and not in the head of the module?

Reply:
The `emails` module imports models from this file (though only for use in type hints), so doing that would lead to a circular import.